### PR TITLE
Add alfred-cool

### PR DESCRIPTION
**Pull request by nguyenvanduocit**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -553,6 +553,7 @@ Non-synced local preferences are stored within `Alfred.alfredpreferences` under
 - [alfred-jira](https://github.com/colinf/alfred-jira) - Convert clipboard text between Markdown and Jira markup
 - [alfred-homebrew](https://github.com/vinkla/alfred-homebrew) - Search for macOS packages with Homebrew
 - [alfred-network-location-switch](https://github.com/abdul/alfred-network-location-switch) - Switch macOS network location
+- [alfred-cool](https://github.com/nguyenvanduocit/alfred-cool) - Find for cool words
```
Comment (SamVerschueren) [inline readme.md]: > Find cool words

You can also add tests easily with [alfy-test](https://github.com/samverschueren/alfy-test).

**Comment by SamVerschueren** [comment]
Thanks for the PR! One small remark :).

File: readme.md (hunk 1 above)
Comment (nguyenvanduocit) [reply readme.md]: Sure, I will

Comment (nguyenvanduocit) [reply readme.md]: @SamVerschueren Done

Comment (SamVerschueren) [reply readme.md]: Can you change the description as well to `Find cool words`? Looks better imo.

Comment (nguyenvanduocit) [reply readme.md]: sure, I am no very good at English.

**Comment by nguyenvanduocit** [comment]
@SamVerschueren Updated

**Pull request merged**